cmd/user-agent/package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@trustbloc-cicd/agent-sdk": "0.1.5-snapshot-3b31dd8",
+    "@trustbloc-cicd/agent-sdk": "0.1.5-snapshot-9ac0a0f",
```

Comment:
Do you need to change it in test package too?

Reply:
https://github.com/trustbloc/edge-agent/commit/6d089e95e2d34abc5ec62baf03a307453a4ee8bc

Reply:
done